src/crimson/os/seastore/btree/fixed_kv_node.h

```diff
@@ -972,6 +972,7 @@ struct FixedKVLeafNode
   get_child_ret_t<LogicalCachedExtent>
   get_logical_child(op_context_t<NODE_KEY> c, uint16_t pos) final {
     auto child = this->children[pos];
+    ceph_assert(!is_reserved_ptr(child));
```

Comment:
OK, `FixedKVLeafNode::get_logical_child()` isn't possible to be called when the child is reserved.

How about also add an `assert(!is_reserved_ptr(child))` to `FixedKVNode::get_child()`?